### PR TITLE
Add config flow to AnthemAv integration

### PR DESCRIPTION
**Pull request by Hyralex**
## Proposed change

Add config flow to Anthem AV integration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/53268

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @Hyralex, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#18572).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/anthemav.markdown [hunk 1]
```diff
@@ -18,21 +21,16 @@ Both [Anthem]'s current and last generation of A/V Receivers and Processors supp
...
-```yaml
-# Example configuration.yaml entry
-media_player:
-  - platform: anthemav
-    host: IP_ADDRESS
-```
+{% include integrations/config_flow.md %}
 
-{% configuration %}
+{% configuration_basic %}
```
Comment (frenck) [inline source/_integrations/anthemav.markdown:33]: A `configuration_basic` block does not use `required`, `type`, etc. It only has `description` at this point.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/anthemav.markdown (hunk 1 above)
Comment (Hyralex) [reply source/_integrations/anthemav.markdown:33]: ah right.  Thanks. I changed it back.

Comment (frenck) [reply source/_integrations/anthemav.markdown:33]: Well that was not the correct move :)

`configuration` is used to describe YAML options, so using `configuration_basic` was the right choice. You just needed to remove the additional unused parameters.

Comment (Hyralex) [reply source/_integrations/anthemav.markdown:33]: Well, that was a friday move 😅 At least I'm learning. Thanks, changing now.

Comment (Hyralex) [reply source/_integrations/anthemav.markdown:33]: alright, I think I got it right this time :)

Comment (frenck) [reply source/_integrations/anthemav.markdown:33]: Yep! Thanks! 🙏

**Review by frenck — APPROVED** [review]
Thanks, @Hyralex 👍

**Pull request merged**